app/_src/gateway-operator/guides/plugin-distribution.md

```diff
@@ -0,0 +1,257 @@
+
+   ```bash
+   docker tag myheader:1.0.0 <YOUR-REGISTRY-ADDRESS>/myheader:1.0.0
+   docker push <YOUR-REGISTRY-ADDRESS>/myheader:1.0.0
+   ```
+
+   The plugin from the example above is available in the public registry (Docker Hub) as `kong/plugin-example:1.0.0` for everyone. That source is used in the following steps.
+
+3. Install the plugin using the `KongPluginInstallation` resource. It makes it available for instances of {{site.base_gateway}} resources.
+
+   ```yaml
+   echo '
+   kind: KongPluginInstallation
+   apiVersion: gateway-operator.konghq.com/v1alpha1
+   metadata:
+     name: custom-plugin-myheader
+   spec:
+     image: kong/plugin-example:1.0.0
+   ' | kubectl apply -f -
+   ```
+
+   Learn more about the `KongPluginInstallation` resource in the [CRD reference documentation](/gateway-operator/{{page.release}}/reference/custom-resources/#kongplugininstallation).
+
+   Verify that the plugin is fetched and available by examining the status of the `KongPluginInstallation` resource:
+
+   ```bash
+   kubectl get kongplugininstallations.gateway-operator.konghq.com -o jsonpath-as-json='{.items[*].status}'
+   ```
+
+   The output should look like this:
+
+   ```json
+   [
+     {
+         "conditions": [
+             {
+                 "lastTransitionTime": "2024-10-09T19:39:39Z",
+                 "message": "plugin successfully saved in cluster as ConfigMap",
+                 "observedGeneration": 1,
+                 "reason": "Ready",
+                 "status": "True",
+                 "type": "Accepted"
+             }
+         ],
+         "underlyingConfigMapName": "custom-plugin-myheader-hnzf9"
+     }
+   ]
+   ```
+
+   in case of problems respective `conditions` or respective resources will provide more information.
+
+   {:.note}
+    > `KongPluginInstalaltion` resource creates a `ConfigMap` with the plugin content. Additional `ConfigMap`s are created when a plugin is referenced by other resources. The operator automatically manages the lifecycle of all these `ConfigMap`s.
+
+4. Make the plugin available in a `Gateway` resource by referencing it in the `spec.pluginsToInstall` field of the `GatewayConfiguration` resource.
+   Plugins can be referenced cross-namespace without any additional configuration.
+
+   ```yaml
+   echo '
+   kind: GatewayConfiguration
+   apiVersion: gateway-operator.konghq.com/v1beta1
+   metadata:
+     name: kong
+     namespace: default
+   spec:
+     dataPlaneOptions:
+       deployment:
+         replicas: 2
+         podTemplateSpec:
+           spec:
+             containers:
+               - name: proxy
+                 image: kong/kong-gateway:{{ site.data.kong_latest_gateway.ce-version }}
```

Comment:
My mistake on suggesting this variable here. `kong/kong-gateway` doesn't ship `MAJOR.MINOR.PATCH` tags.

```suggestion
                 image: kong/kong-gateway:{{ site.data.kong_latest_gateway.ee-version }}
```